Coucat/Pokedex
# POK-15-firebase-deploy

# Pull Request

## Description

### Ticket impemented #[POK-15](https://coucat.atlassian.net/browse/POK-15)

[//]:<> (You can see the number of ticket in Jira in the top of task, replace the ticketNumber in the space and the number in the link)
### What did you implement it?

[//]:<> (Mark with an 'x' or a space)

- [ ] Bug Fix (Non-breaking change wich fixes an issue)
- [x] New Feature (Non-breaking change wich adds functionality)


### How did you implement it?
[//]:<> (In this section describe all things that you made)

Project connection with Firebase CLI.
Firebase Hosting configuration:
- Firebase feature : Hosting configure files for Firebase Hosting
- Project selected: Coucat
- Hosting setup: 
        - Public directory: /build
        - Single-page app: Yes
        - Automatic builds: Yes
        - Overwrite: No
- Connect with your github account 
- Workflow: coucat/pokedex
- Run a build script: Yes
- Script: 
```sh
npm ci && npm run build
```
- Automatic deployment: Yes
- GitHub branch associated: main
## How can we test this ticket?
[//]:<> (Write in this part all the commands that you can use to test your code)
In first time you need build the project with
```sh
yarn build 
```
After you need do some changes in your branch and do a push with a PR
```sh
git add -A
git commit -m "text of changes"
git push origin -nameBranch-
```
You can go to the [ACTIONS](https://github.com/Coucat/Pokedex/actions/new) section on github and see the deploy.

## Checklist
[//]:<> (Mark with a 'x' or a space )
